docs/modules/ROOT/partials/proc-registry-https-in-cluster.adoc

```diff
@@ -17,103 +18,68 @@ You can skip this step if you are using your own certificates.
+--
+or create the secret using the following command:
 
-.. Add a volume mount:
-+
-[source,yaml]
-----
-volumeMounts:
-  - name: registry-keystore-secret-volume
-    mountPath: /etc/registry-keystore
-    readOnly: true
-----
-
-.. Add `JAVA_OPTIONS` and `KEYSTORE_PASSWORD` environment variables:
-+
-[source,yaml]
-----
-- name: KEYSTORE_PASSWORD
-  valueFrom:
-    secretKeyRef:
-      name: registry-keystore
-      key: password
-- name: JAVA_OPTIONS
-    value: >-
-     -Dquarkus.http.ssl.certificate.key-store-file=/etc/registry-keystore/keystore.jks
-     -Dquarkus.http.ssl.certificate.key-store-file-type=jks
-     -Dquarkus.http.ssl.certificate.key-store-password=$(KEYSTORE_PASSWORD)
-----
-+
-NOTE: Order is important when using string interpolation.
-
-.. Enable the HTTPS port:
-+
-[source,yaml]
+[source,bash]
 ----
-ports:
-  - containerPort: 8080
-    protocol: TCP
-  - containerPort: 8443
-    protocol: TCP
+oc create secret generic https-cert-secret --from-file=tls.key --from-file=tls.crt
 ----
+--
 
-. Edit the *Service* resource of the {registry} instance.
-You can find the correct name in a status field of the {operator}.
+. Edit the `spec.configuration.security.https` section of the `ApicurioRegistry` CR for your {registry} deployment, for example:
 +
 [source,yaml]
 ----
-ports:
-  - name: http
-    protocol: TCP
-    port: 8080
-    targetPort: 8080
-  - name: https
-    protocol: TCP
-    port: 8443
-    targetPort: 8443
+apiVersion: registry.apicur.io/v1
+kind: ApicurioRegistry
+metadata:
+  name: example-apicurioregistry
+spec:
+  configuration:
+    # ...
+    security:
+      https:
+        secretName: https-cert-secret
 ----
 
 . Verify that the connection is working:
 .. Connect into a pod on the cluster using SSH (you can use the {registry} pod):
 +
 [source,bash]
 ----
-oc rsh -n default example-apicurioregistry-deployment-vx28s-4-lmtqb
+oc rsh example-apicurioregistry-deployment-6f788db977-2wzpw
 ----
 
 .. Find the cluster IP of the {registry} pod from the *Service* resource (see the *Location* column in the web console).
 Afterwards, execute a test request (we are using self-signed certificate, so an insecure flag is required):
 +
 [source,bash]
 ----
-curl -k https://172.30.209.198:8443/health
-[...]
+curl -k https://172.30.230.78:8443/health
 ----
+
+NOTE: In the Kubernetes secret containing the HTTPS certificate and key, the names `tls.crt` and `tls.key` must be used for the provided values.
+This is currently not configurable.
+
+.Disabling HTTP
+If you have enabled HTTPS using the procedure above, you can also disable the default HTTP connection by setting the `spec.security.https.disableHttp` to `true`.
+This will remove the HTTP port 8080 from the {registry} pod container, `Service`, and the `NetworkPolicy`  (if present).
+Importantly, `Ingress` will also be removed, since the {operator} currently does not support configuring HTTPS in `Ingress`.
+Users must create an `Ingress` for HTTPS connections manually.
```

Comment:
```suggestion
This removes the HTTP port 8080 from the {registry} pod container, `Service`, and the `NetworkPolicy`  (if present).
Importantly, `Ingress` is also removed because the {operator} currently does not support configuring HTTPS in `Ingress`.
Users must create an `Ingress` for HTTPS connections manually.
```